docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,29 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.7.2 - July 4, 2024
+
+### New features
+
+- Support for running [Assess Migration](../migrate/assess-migration/) against Oracle source database.
```

Comment:
minor re-phrasing:  
support for Oracle Migration Assessment using assess-migration command